capture/genBitmaps.js

```diff
@@ -186,7 +208,7 @@ function processScenario (casper, scenario, scenarioOrVariantLabel, scenarioLabe
           fileName = fileName + '.png';
         }
 
-        var referenceFilePath = bitmapsReferencePath + '/' + fileName;
+        var referenceFilePath = bitmapsReferencePath + '/' + fileName.replace(/__n\d*__\d*/, '');
```

Comment:
See filenameTemplate string declared earlier in the script.  

I think we should try to not modify name segments named by other features.  This keeps features isolated.  The new filename template should look like this...

{configId}_{scenarioLabel}_{selectorIndex}_{selectorLabel}_{viewportIndex}_{viewportLabel}{expansionIndex};

...and for an element which repeats 3 times expansionIndex should equal '', '__n1', '__n2'.

Does that help?